src/main/kotlin/dev/supergrecko/kllvm/core/typedefs/Builder.kt

```diff
@@ -53,7 +55,38 @@ public class Builder internal constructor(internal val llvmBuilder: LLVMBuilderR
         // TODO: Test
         LLVM.LLVMInsertIntoBuilderWithName(getUnderlyingRef(), instruction.getUnderlyingReference(), name)
     }
+    /**
+     * Create a function call passing in |args| and binding the result into
+     * variable |resultName|. Result discarded if no resultName supplied.
+     * @see LLVM.LLVMBuildCall
+     */
+    public fun buildCall(
+        function: Value,
+        args: List<Value>,
+        resultName: String? = null
+    ) : InstructionValue /* TODO: Replace with CallInstruction when type is created  */ {
```

Comment:
Prefer placing the TODO comment inside the doccomment

Reply:
And fixed